zellij.hcl

```diff
@@ -0,0 +1,26 @@
+description = "A terminal workspace with batteries included."
+binaries = ["zellij"]
+test = "zellij --version"
+
+linux {
+  source = "https://github.com/zellij-org/zellij/releases/download/v${version}/zellij-no-web-${xarch}-unknown-linux-musl.tar.gz"
```

Comment:
`xarch` var was a nice finding... At first glance, I see many packages that would benefit of using it:
https://github.com/search?q=repo%3Acashapp%2Fhermit-packages%20arch_&type=code

Reply:
You'll find that mostly `arch_` is used for packages that don't map correctly to xarch.